src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_dps_commands.py

```diff
@@ -23,10 +23,14 @@ def test_dps_lifecycle(self, group_name, group_location):
                          self.check('sku.name', 'S1')])
 
         # Create DPS
-        self.cmd('az iot dps create -g {} -n {}'.format(group_name, dps_name), checks=[
+        tags = "key1=value1 key2=value2"
+        dps = self.cmd('az iot dps create -g {} -n {} --tags {}'.format(group_name, dps_name, tags), checks=[
             self.check('name', dps_name),
-            self.check('location', group_location)
-        ])
+            self.check('location', group_location),
+            self.check('tags', {'key1': 'value1', 'key2': 'value2'})
+        ]).get_output_in_json()
+
+        assert len(dps['tags']) == 2
```

Comment:
Because the tag itself is checked, you can delete this check
```suggestion
```

Reply:
Fixed